.all-contributorsrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "files": ["readme.md"],
```

Comment:
We can put another file in the array here, so if we want to move this out of the readme into a separate file, we can do that after we merge the first PR from the all contributors bot itself.